crypto/poly1305/asm/poly1305-armv4.pl

```diff
@@ -677,6 +677,9 @@
 	cmp	$len,#64
 	bhs	.Lenter_neon
 	tst	ip,ip			@ is_base2_26?
+#ifdef	__thumb2__
+	it	eq
+#endif
 	beq	poly1305_blocks
```

Comment:
I'd like to suggest (which is basically euphemism for insist:-) on removing this relocation altogether. I mean suggested change replaces one relocation type with another, but why have relocation at all if we can get rid of it? So drop #ifdef, `beq .Lpoly1305_blocks` and add `.Lpoly1305_blocks:` right after `poly1305_blocks:`.

Reply:
Done.